lapack/gonum/dsytrd.go

```diff
@@ -75,11 +68,16 @@ func (impl Implementation) Dsytrd(uplo blas.Uplo, n int, a []float64, lda int, d
 		panic(shortWork)
 	}
 
+	// Quick return if possible.
 	if n == 0 {
 		work[0] = 1
 		return
 	}
 
+	opts := "U"
```

Comment:
Didn't we change these so that `blas.Lower = 'L'` and `blas.Upper = 'U'`? So `nb := impl.Ilaenv(1, "DSYTRD", string(uplo), n, -1, -1, -1)`. Or is this to save the alloc?

Reply:
I just used what was here before. Now replaced with your suggestion.